polaris-migrator/src/utilities/testUtils.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable jest/no-export, jest/valid-title, @typescript-eslint/no-var-requires */
```

Comment:
Dragging a comment over from the old PR: https://github.com/Shopify/polaris/pull/6701#discussion_r936982008

jscodeshift exposes a `runTest` utility https://github.com/Shopify/web/blob/main/tooling/codemods/tests/remove-autobind.test.js#L4 (though it lacks types, so you'll need to [add those yourself](https://github.com/Shopify/web/blob/main/tooling/typescript/modules.d.ts#L73-L81))

It seems to do the same as everything you're doing here. Is it worth leveraging that already existing utility instead of writing our own?

Reply:
I like the idea of not maintaining our own version of such a similar function. The `runTest` function from `jscodeshift` does make some assumptions about the directory structure that I didn't prefer (ex: needing to use the `__tests__` and `__testfixtures__` conventions).

I think having our own check function allows us to simplify the function arguments and create our own conventions. I like having the single `tests` directory where all tests and fixtures can live for a specific migration.

Reply:
Sounds good to me 👍